plugins/models/vae.py

```diff
@@ -0,0 +1,71 @@
+import keras
+from keras import backend as K
+from keras import objectives
+from keras.layers import *
+from keras.models import Model, Sequential
+from keras.optimizers import Adam
+
+from deepprofiler.learning.model import DeepProfilerModel
```

Comment:
Excellent job with the VAE! Same comments here. Let's try to add a link to a paper, and also add some basic comments.